### PR TITLE
Ddcyls 3736

### PR DESCRIPTION
**Pull request by Michael-RJ**
[Your pull request description here]

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [ ] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [ ] Requires acceptance test run.
- [ ] Appropriate labels added.
- [ ] RELEASE NOTES ADDED.

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -1,25 +1,22 @@
 import sbt._
 
 object AppDependencies {
-  import play.core.PlayVersion
 
-  val compile = Seq(
+  val compile: Seq[ModuleID] = Seq(
     play.sbt.PlayImport.ws,
-    "uk.gov.hmrc"       %% "bootstrap-frontend-play-28"     % "7.13.0",
+    "uk.gov.hmrc"       %% "bootstrap-frontend-play-28"     % "8.4.0",
```
Comment (jpatton638) [inline project/AppDependencies.scala:7]: You could make assign the version number of bootstrap to a value and use this for both the app and test dependencies for bootstrap to ensure they are always the same version

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -1,25 +1,22 @@
...
 
-  val test = Seq(
-    "uk.gov.hmrc"                 %% "bootstrap-test-play-28"  % "7.13.0",
-    "org.scalatest"               %% "scalatest"               % "3.0.9",
-    "org.scalatestplus"           %% "scalacheck-1-17"         % "3.2.16.0",
-    "org.mockito"                 %% "mockito-scala-scalatest" % "1.17.12",
-    "org.scalacheck"              %% "scalacheck"              % "1.15.4"
+  val test: Seq[ModuleID] = Seq(
+    "uk.gov.hmrc"                 %% "bootstrap-test-play-28"  % "8.4.0",
+    "org.scalatest"               %% "scalatest"               % "3.2.17",
```
Comment (jpatton638) [inline project/AppDependencies.scala:15]: You can remove scalatest as bootstrap-test pulls it in

File: project/AppDependencies.scala [hunk 3]
```diff
@@ -1,25 +1,22 @@
...
+    "org.scalatestplus"           %% "scalacheck-1-17"         % "3.2.17.0",
+    "org.mockito"                 %% "mockito-scala-scalatest" % "1.17.30",
+    "org.scalacheck"              %% "scalacheck"              % "1.17.0",
```
Comment (jpatton638) [inline project/AppDependencies.scala:18]: Not sure if this is needed if you have `scalacheck-1-17` pulled in

File: project/AppDependencies.scala (hunk 1 above)
Comment (Michael-RJ) [reply project/AppDependencies.scala]: Done.

File: project/AppDependencies.scala (hunk 2 above)
Comment (Michael-RJ) [reply project/AppDependencies.scala]: Scalatest removed.

File: project/AppDependencies.scala (hunk 3 above)
Comment (Michael-RJ) [reply project/AppDependencies.scala]: Tried to remove some of these dependencies but caused several failures. Was only able to remove "scalatest" and "scalacheck" dependencies.

Comment (jpatton638) [reply project/AppDependencies.scala]: That's great, those were the ones I was talking about removing, the others are still needed, sorry for the confusion!

**Review by jpatton638 — APPROVED** [review]

**Pull request merged**